avaframe/log2Report/generateCompareReport.py

```diff
@@ -20,7 +21,7 @@ def copyQuickPlots(avaName, testName, outDir, plotListRep, rel=''):
 
     plotPaths = {}
     for key in plotListRep:
-        plotName = os.path.join(outDir, '%s_%s_%s.png' % (testName, rel, key))
+        plotName = pathlib.Path(outDir, ('%s_%s_%s.png' % (testName, rel, key)))
```

Comment:
mh, if outDir is converted to a pathlib path, then one could write this a bit cleaner without the pathlib.Path necessary?

Reply:
I introduced a check if outDir is pathlibe PurePath if not outside of the for loop make it a pathlib.Path same for copyAimecPlots